internal/pkg/diff/diff_test.go

```diff
@@ -20,10 +20,11 @@ func TestDiffOnlyInLocal(t *testing.T) {
 	t.Parallel()
 	projectState := createProjectState(t)
 	branchKey := model.BranchKey{Id: 123}
-	branch := &model.Branch{BranchKey: branchKey}
-	branchState, err := projectState.CreateFrom(&model.BranchManifest{BranchKey: branchKey})
-	assert.NoError(t, err)
-	branchState.SetLocalState(branch)
+	branchState := &model.BranchState{
+		BranchManifest: &model.BranchManifest{BranchKey: branchKey},
+		Local:          &model.Branch{BranchKey: branchKey},
+	}
+	assert.NoError(t, projectState.Set(branchState))
```

Comment:
Dobre to vidno na tomto diffe, namiesto viacerych funkcii, iba  jedna struktura a `Set`.